<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Date:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leDate"/>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Title:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leTitle"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Author:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leAuthor"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cantarell'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>678</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionCredentials"/>
    <addaction name="actionConnection"/>
    <addaction name="actionLocal_Storage"/>
    <addaction name="actionRemote_Storage"/>
    <addaction name="actionEditor"/>
   </widget>
   <widget class="QMenu" name="menuPost">
    <property name="title">
     <string>Post</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_Remote"/>
    <addaction name="actionSave"/>
    <addaction name="actionUpload_remote"/>
    <addaction name="actionSync"/>
   </widget>
   <addaction name="menuPost"/>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionConnection">
   <property name="text">
    <string>Connection</string>
   </property>
  </action>
  <action name="actionLocal_Storage">
   <property name="text">
    <string>Local Storage</string>
   </property>
  </action>
  <action name="actionRemote_Storage">
   <property name="text">
    <string>Remote Storage</string>
   </property>
  </action>
  <action name="actionEditor">
   <property name="text">
    <string>Editor</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save (local)</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open Local</string>
   </property>
  </action>
  <action name="actionOpen_Remote">
   <property name="text">
    <string>Open Remote</string>
   </property>
  </action>
  <action name="actionUpload_remote">
   <property name="text">
    <string>Upload (remote)</string>
   </property>
  </action>
  <action name="actionSync">
   <property name="text">
    <string>Sync</string>
   </property>
  </action>
  <action name="actionCredentials">
   <property name="text">
    <string>Credentials</string>
   </property>
  </action>
  <action name="actionWorking_Directory">
   <property name="text">
    <string>Working Directory</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
